prov/gni/test/rdm_sr.c

```diff
@@ -474,6 +497,33 @@ void rdm_sr_xfer_for_each_size(void (*xfer)(int len), int slen, int elen)
 	}
 }
 
+static inline void rdm_sr_check_err_cqe(struct fi_cq_err_entry *cqe, void *ctx,
+					uint64_t flags, void *addr, size_t len,
```

Comment:
There seems to be something missing from this function. How do you verify that you get the right data back from the err_cqe? Does the err_cqe have all the proper fields filled out for a source addr error? If so, how have you verified it?